snapcraft.yaml.template

```diff
@@ -0,0 +1,22 @@
+name: jq
+version: @JQ_VERSION@
+summary: jq is a lightweight and flexible command-line JSON processor.
+description: jq is like sed for JSON data - you can use it to slice and filter and map and transform structured data with the same ease that sed, awk, grep and friends let you play with text.
```

Comment:
The long line can be avoided with YAML syntax:

    description: |
      jq is like sed for JSON data - you can use it to slice and filter and map
      and transform structured data with the same ease that sed, awk, grep and
      friends let you play with text.